index.js

```diff
@@ -188,6 +181,14 @@ class Item {
     droppedItem = playerInv.splice(droppedItemIndex, 1).toString();
     return `You dropped the ${this.name}`;
   }
+
```

Comment:
another example of line by line:
//this method can be called on an item to determine if it can be read.